clients/search-component/package.json

```diff
@@ -52,6 +52,7 @@
     "@r2wc/react-to-web-component": "^2.0.3",
     "@thumbmarkjs/thumbmarkjs": "^0.14.8",
     "react-markdown": "^9.0.1",
+    "react-snap-carousel": "^0.5.0",
```

Comment:
How big is this package? Really tempted to say we should build this completely from scratch.

Reply:
This package is ~3.8kb.

Reply:
Sounds good to me then